app/controllers/videos_controller.rb

```diff
@@ -0,0 +1,15 @@
+class VideosController < ApplicationController
+  before_action :require_user
+  def index
+    @videos = Video.all
+    @categories = Category.all
+  end
```

Comment:
Different actions should be usually split by an empty space line. Gives better impression of an organized code, plus is more readable.